index.html

```diff
@@ -2,9 +2,113 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Domein.shop</title>
+    <link rel="stylesheet" href="styles/styles.css">
 </head>
 <body>
-
+<header>
+    <div class="header-items">
+        <h1>Domein.shop</h1>
+        <p>Jouw domeinnaam registreren in slechts 1 minuut!</p>
+        <form onsubmit="JS">
+            <label for="domein">
+                <input class="header-input" type="text" id="domein" name="domein"
+                       placeholder="Is jouw domein nog vrij?"/>
+            </label>
+            <button class="header-button" type="button">
```

Comment:
Omdat de button in een form staat zou je hier type="submit" meer gepast zijn
```suggestion
            <button class="header-button" type="submit">
```